app/src/main/java/edu/grinnell/appdev/grinnelldirectory/Activity/LoginActivity.java

```diff
@@ -83,9 +85,7 @@ void signIn(View view) {
      */
     @BindString(R.string.server_failure) String serverFailure;
     @Override public void onServerFailure(String fail_message) {
-        AlertDialog.Builder builder = new AlertDialog.Builder(this);
-        builder.setMessage(serverFailure + fail_message);
-        builder.show();
+        showAlert(serverFailure + fail_message);
```

Comment:
I think having two parameters for this method will be cleaner. You will just have to do the concatenation of those two in `showAlert(...)`. One advantage of doing it your way is that we could easily reuse `showAlert(...)` if it were public but it's a `private` method so you might as well just go ahead and do the concat. in `showAlert(...)`